dashboard/app/views/levels/editors/_gamelab_jr.html.haml

```diff
@@ -0,0 +1,30 @@
+.field
+  = f.label :custom_blocks, 'Custom block defintions'
+  %p
+    Add some custom JSON block definitions
+    <a href="#" onClick="toggleBlock('custom_blocks_more'); return false">Read more...</a>
+  #custom_blocks_more{style: 'display: none'}
+    %p
+      Here is an example.
+    %pre
+      :preserve
+        {
+          "func": "isDestroyed",
+          "args": [
+            { "name": "SPRITE" }
+          ],
+          "blockText": "{SPRITE} is destroyed?",
+          "returnType": "Boolean",
```

Comment:
Extra comma here is making the example JSON un-parsable.

Reply:
Added [jsonic](https://www.npmjs.com/package/jsonic) parsing to make it less annoying to hand-craft the JSON here. If you don't quote keys or have trailing commas, the editor will automatically fix it up for you when you unfocus the textbox.